synapse/tools/genpkg.py

```diff
@@ -55,6 +56,34 @@ def loadPkgProto(path, opticdir=None):
     protodir = os.path.dirname(full)
     pkgname = pkgdef.get('name')
 
+    logodef = pkgdef.get('logo')
+    if logodef is not None:
+
+        path = logodef.pop('path', None)
+
+        if path is not None:
+            with s_common.reqfile(protodir, path) as fd:
+                logodef['file'] = base64.b64encode(fd.read()).decode()
+
```

Comment:
May need a check here to verify that 'file' is present? Not sure what the expected result would be if for some reason 'mime' was specified but no 'path' or 'file'.